everviz/util.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+
+def default_config_file():
```

Comment:
Should this be a function or a variable, `DEFAULT_CONFIG_FILE_NAME`?

Reply:
Why should that be a global constant?

I know a lot of out projects are using global constants but they are highly discouraged in a lot of other programming languages.

Reply:
Well, as it´s just a public function returning a string, I don´t quite see the distinction as long as it´s immutable. But I might very well be wrong.

Reply:
The reason I don't really like all the "constants"  are all around the place is that they are not really constants, they are constants by convention and there is no support in the language for them. You can really easily change them while a script is running which is not very constant like. 

But  I can find multiple sources that are suggesting this as a good and normal practice when it comes to "constants" which means I might very well be wrong too.

I will change this such that it is consistent with "constant" variables.